CONTRIBUTION.md

```diff
@@ -34,17 +36,26 @@ Please follow this guidelines when contributing to the project.
 
 ## Rules
 
-| field         | description | required/optional |
-| ------------- | ----------- | ----------------- |
-| name          |             | required          |
-| rollNo        |             | required          |
-| degree        |             | required          |
-| branch        |             | required          |
-| yearOfPassing |             | required          |
-| github        |             | required          |
-| linkedin      |             | optional          |
-| blog          |             | optional          |
-| portfolio     |             | optional          |
-| languages     |             | optional          |
+| field         | description                                          | required/optional |
+| ------------- | ---------------------------------------------------- | ----------------- |
+| name          | Your name                                            | required          |
+| rollNo        | Your JKLU Roll no. (eg. 2020btechcse0xx)             | required          |
+| degree        | The degree you are pursuing (B.Tech., B.Des. etc.)   | required          |
+| branch        | Your branch (eg. CSE, EEE, ME etc.)                  | required          |
+| yearOfPassing | Your Graduation year                                 | required          |
+| github        | Github username                                      | required          |
+| linkedin      | Link to your linkedIn profile                        | optional          |
+| blog          | Link to your blog                                    | optional          |
+| portfolio     | Link to your portfolio                               | optional          |
+| languages     | The programming languages that you are familiar with | optional          |
 
 ---
+
+## Register for Hacktoberfest
```

Comment:
add this part in readme

Reply:
Steps to register for hacktoberfest has already been added